app/frontend/src/internationalization/trans/chefs/es/es.json

```diff
@@ -692,7 +690,7 @@
     "no": "No",
     "failedResSubmissn": "Respuesta fallida desde el punto final de envío. Código de respuesta: {estado}",
     "errSubmittingForm": "Ocurrió un error al enviar este formulario",
-    "errorSavingFile": "Error al guardar archivos. Nombre de archivo: {fileNname}. Error: {error}",
+    "errorSavingFile": "Error al guardar archivos. Nombre de archivo: {nombre de archivo}. Error: {error}",
```

Comment:
In the other files `fileNname` is corrected to `fileName` but I think this translation will break the parameter.